app/helpers/exam_registration_helper.rb

```diff
@@ -6,8 +6,4 @@ def exam_registration_view
       { icon: :info_circle, class: :info, t: :exam_registration_explanation_html }
     end
   end
-
-  def current_time_zone_html
-    %Q{(<span class="select-date-timezone">#{Organization.current&.time_zone}</span>)}.html_safe
```

Comment:
I removed the `&` since this method will never return `nil`. The nil-version is `safe_current`, which returns nil instead of throwing an exception.